frontend/app/src/bounties/__mock__/mockLocalStorage.utils.ts

```diff
@@ -0,0 +1,29 @@
+const mockLocalStorage = (() => {

```

Comment:
is this the same as in `./src/__tests__/__mocks__/localStorage.ts`

it might make sense to use that one

Reply:
Okay, I've done that. Thanks Kevin